profitbricks/src/main/java/org/jclouds/profitbricks/compute/function/ImageToImage.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.profitbricks.compute.function;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static org.jclouds.profitbricks.compute.function.DataCenterToLocation.mapLocation;
+
+import java.util.regex.Pattern;
+
+import org.jclouds.compute.domain.Image;
+import org.jclouds.compute.domain.ImageBuilder;
+import org.jclouds.compute.domain.OperatingSystem;
+import org.jclouds.compute.domain.OsFamily;
+
+import com.google.common.base.Function;
+
+public class ImageToImage implements Function< org.jclouds.profitbricks.domain.Image, Image> {
+
+   private static final Pattern HAS_NUMBERS = Pattern.compile(".*\\d+.*");
+
+   @Override
+   public Image apply(org.jclouds.profitbricks.domain.Image from) {
+      checkNotNull(from, "Null image");
+
+      String desc = from.name();
+      OsFamily osFamily = parseOsFamily(desc);
+
+      OperatingSystem os = OperatingSystem.builder()
+	      .description(osFamily.value())
+	      .family(osFamily)
+	      .version(parseVersion(desc))
+	      .is64Bit(is64Bit(desc))
+	      .build();
+
+      return new ImageBuilder()
+	      .ids(from.id())
+	      .name(desc)
+	      .location(mapLocation(from.location()))
```

Comment:
Hmm. I couldn't tell if it it qualifies to the __available in all regions__ statement. Let me describe it instead.

So given is an image called `Ubuntu-14.04-LTS-server-2015-03-01`. This exists in [Location](https://github.com/jclouds/jclouds-labs/pull/145/files#diff-8dba16965654b3b80ba6cc57f45b8278L19)s: `de/fra`, `de/fkb` but with different `id`s. There's also a _similar_ image called `Ubuntu-14.04-LTS-server-2015-03-02` in `us/las` and `us/lasdev` also different IDs. Does this qualify to the statement? (Does this present an issue?)

Also, I thought of `LocationScope` as a hierarchy. So `Provider` -> `Region` -> `Zone`, etc., and so I mapped it like `Location`(=`Provider`) -> `DataCenter`(`=Region`). Correct me if I assumed incorrectly